docs/ipxe_home_server.md

```diff
@@ -0,0 +1,57 @@
+# iPXE on home server #
+
+This guide is about setting up an iPXE-tftp server on home server machine.
+
+## Installation ##
+
+1. Server should be running inside a Docker container therefore docker is
+required. To install docker on Debian go [here](https://docs.docker.com/engine/installation/linux/debian/).
+
+2. To build environment and install ipxe server go to [this](https://github.com/3mdeb/pxe-server)
+address and follow instructions. I belive that according to FHS, pxe-server
+should be cloned to `/opt/pxe-tftpd/`. Note that this catalog is used later. It
+can be changed but this requires changes in `ipxe-tftp.service` (if used).
```

Comment:
@leszekbulawa something is wrong since I see different service name:

```
[17:50:40] pietrushnic:pxe-server git:(master) $ sudo service ipxe.tftp status            
● ipxe.tftp.service - tftp pxe server container
   Loaded: loaded (/lib/systemd/system/ipxe.tftp.service; disabled)
   Active: inactive (dead)
```